README.md

```diff
@@ -84,9 +134,7 @@ To create an image based on Ubuntu Xenial for Raspberry Pi 2 with [htop](https:/
 $ sudo env DEVICE=rpi-2-b OS=ubuntu-xenial-armhf INCLUDES=htop,mc ./pieman.sh
 ```
 
-The built images will be located in `build/<project name>`.
-
-You can also find details on `DEVICE`, `INCLUDES`, `OS` and other environment variables (called parameters) which help customizing images in the Documentation section.
+The built images will be located in `build/<project name>`. By the way, you can specify the name of your project via the `PROJECT_NAME` environment variable. You can find details on `DEVICE`, `INCLUDES`, `OS`, `PROJECT_NAME` and other environment variables (called parameters) which help customizing images in the Documentation section.
```

Comment:
What if we add PROJECT_NAME=myimage.img to the example above? After that we can remove "By the way, you can specify the name of your project via the `PROJECT_NAME` environment variable."

Reply:
The main idea is that `PROJECT_NAME` is unique by default. So, each time you run the script, you get a new image file. `PROJECT_NAME=myimage.img` probably solves some problems, but it may cause other ones.
Our goal is not to make the documentation as short as possible, so I don't see any problems with

> By the way, you can specify the name of your project via the `PROJECT_NAME` environment variable. You can find details on `DEVICE`, `INCLUDES`, `OS`, `PROJECT_NAME` and other environment variables (called parameters) which help customizing images in the Documentation section.